batch/batch/driver/instance_pool.py

```diff
@@ -97,13 +156,13 @@ async def update_zones(self):
                 for q in r['quotas']
             }
 
-            remaining = quota_remaining['PREEMPTIBLE_CPUS'] // self.worker_cores
+            remaining = quota_remaining['PREEMPTIBLE_CPUS'] / self.worker_cores
```

Comment:
I changed these because the `random.choices` can handle floats.  Seems more accurate.